src/sub-blocks/Content/Content.tsx

```diff
@@ -72,6 +75,7 @@ const Content = (props: ContentBlockProps & ClassNameProps) => {
                     />
                 </div>
             )}
+            {contentList && <ContentList contentList={contentList} size={size} />}
```

Comment:
maybe just check the length of the array and not even render the component if it's empty?